cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml

```diff
@@ -2,8 +2,6 @@ title: Corpora schema version 5.X.X
 type: anndata
 # If sparsity of any expression matrix is greater than this and not csr sparse matrix, then there will be warning.
 sparsity: 0.5
-# If the R array will exceed this number in size, then Seurat conversion will fail
-max_size_for_seurat: 2147483647  # 2^31 - 1 (max value for 4-byte signed int)
```

Comment:
it may be best to hold off on this change until we confirm from @brianraymor we want to remove this from the schema definition

Reply:
That language is not part of the schema definition. And all references to Seurat/R will be removed from 5.3.